src/Layout/Header/Tools/__tests__/index.spec.tsx

```diff
@@ -147,8 +148,8 @@ describe('Page header tools', () => {
     const infoButton = screen.getByRole('button', { name: 'info button' });
     fireEvent.click(infoButton);
 
-    const communityItem = screen.getByRole('menuitem', { name: /Community/i });
-    fireEvent.click(communityItem);
+    const helpItem = screen.getByRole('menuitem', { name: /Help/i });
```

Comment:
I did not manage to set that value from the variable quickly, as workaround hard-coded value is here

Reply:
```suggestion
    const helpItem = screen.getByRole('menuitem', { name: new RegExp(helpTitle, 'i') });
```